aii-core/src/main/perl/Shellfe.pm

```diff
@@ -30,8 +30,9 @@ use File::Path qw(mkpath rmtree);
 use File::Basename qw(basename);
 use DB_File;
 use Readonly;
-
+use Parallel::ForkManager 0.7.6;
```

Comment:
As for me, we don't need AII to be able to run on a SL5 machine (SL5 is almost EOL if I am right). Only the ability to manage a SL5 node is needed.